tools/phpcs/phpcs.ruleset.xml

```diff
@@ -44,22 +44,6 @@
 		<exclude-pattern>tests/*</exclude-pattern>
 	</rule>
 
-	<!-- Do not apply filename rules for unit tests and object cache -->
-	<rule ref="WordPress.Files.FileName.NotHyphenatedLowercase">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
```

Comment:
Autoloading test helpers with Composer has been removed so PSR4 file naming is no longer, required as it is not ideal to autoload test helpers for multiple plugins from a single location. In most cases, PHPUnit's file loader will handle this effectively.

Reply:
Where were the autoloading test helpers removed?

Reply:
Moved them to their plugin tests/data directories.